test-app/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "test-app/tests/*": [
+        "tests/*"
+      ],
+      "test-app/*": [
+        "app/*"
+      ],
+      "*": [
+        "types/*"
+      ]
+    },
+    "moduleResolution": "node16",
```

Comment:
I'm not sure why the `test-app` package now needs `'node16'`. Before the changes, the value had been set to `'node'` by `@tsconfig/ember`.

I based the change on https://github.com/simonihmig/ember-stargate/commit/f55c61c9f613475914569512120e12918e0f38bc.

Reply:
Related: https://github.com/embroider-build/addon-blueprint/pull/42/files#r1000570746